src/routes/PublicLayoutRouter/PublicLayoutRouter.scss

```diff
@@ -16,6 +16,13 @@
   }
 }
 
+.wrapper {
```

Comment:
this seems like a significant enough thing to require some more testing for all use cases, i will do that

Reply:
Yep. This is returning the public layout to more similar to the auth layout, since those diverged with routing changes. There are still some differences styling things that are a little off but didn't want to get any more sidetracked from the scope of the ticket that this.